apps/server/src/modules/school/uc/school.uc.ts

```diff
@@ -27,6 +27,18 @@ export class SchoolUc {
 			action: Actions.read,
 			requiredPermissions: [Permission.SCHOOL_EDIT],
 		});
+		const school: SchoolDO = await this.schoolService.getSchoolById(schoolId);
```

Comment:
SchoolDO should not be needed.

Reply:
You are right it is not needed but it is recommended.

It helps make the code clearer and easier to read as it gives the reader a clear indication of what type the variable is. This is particularly important in TypeScript, since TypeScript is a statically typed language, which allows the compiler to catch type errors early on.